changelog.txt

```diff
@@ -58,6 +58,7 @@
 * Fix - Correctly handle countries without states when using the express payment methods
 * Update - Include extension data from block checkout when submitting an express checkout order
 * Fix - Add order locking when processing payment redirects, to mitigate cases of double status updates
+* Fix - Restore Express Checkout buttons on “order-pay” pages by registering the scripts early
```

Comment:
As a nit-pick, it feels like #4446 doesn't register the scripts earlier per se, it makes sure the scripts are registered before using them.